src/rubymotion-apple.h

```diff
@@ -29,6 +29,7 @@ VALUE rb_uint2big(VALUE);
 long rb_num2int(VALUE);
 #define FIX2INT(x) ((int)FIX2LONG(x))
 #define NUM2INT(x) (FIXNUM_P(x) ? FIX2INT(x) : rb_num2int((VALUE)x))
+#define SSIZET2NUM(v) LONG2NUM(v)
```

Comment:
Could we rename this `SSIZE2NUM` instead? (basically drop the `T` from the name)